platform/security/idp/security-idp-client/src/test/java/org/codice/ddf/security/idp/client/AssertionConsumerServiceTest.java

```diff
@@ -415,23 +457,10 @@ public void testProcessSamlResponseEmptySamlResponse() throws Exception {
         is(HttpStatus.SC_INTERNAL_SERVER_ERROR));
   }
 
-  @Test
-  public void testProcessSamlResponseLoginFail() throws Exception {
-    doThrow(AuthenticationFailureException.class)
-        .when(loginFilter)
-        .doFilter(any(ServletRequest.class), isNull(ServletResponse.class), any(FilterChain.class));
-    Response response =
-        assertionConsumerService.processSamlResponse(this.cannedResponse, RELAY_STATE_VAL, false);
-    assertThat(
-        "The http response was not 500 SEVER ERROR",
-        response.getStatus(),
-        is(HttpStatus.SC_INTERNAL_SERVER_ERROR));
-  }
-
```

Comment:
📘 Removed because the call to `AssertinConsumerService` doesn't go through filters.